addons/medical_vitals/functions/fnc_updateOxygen.sqf

```diff
@@ -56,14 +61,14 @@ private _oxygenSaturation = (IDEAL_PPO2 min _po2) / IDEAL_PPO2;
 private _negativeChange = BASE_OXYGEN_USE;
 
 // Fatigue will demand more oxygen
-if (missionNamespace getVariable [QEGVAR(advanced_fatigue,enabled), false]) then {
+if (_unit == player && {missionNamespace getVariable [QEGVAR(advanced_fatigue,enabled), false]}) then {
```

Comment:
`player` deliberately? IIRC Adv Fatigue also supports remote controlled AI

Reply:
Ah, yes it does